src/tools/miri/src/borrow_tracker/stacked_borrows/mod.rs

```diff
@@ -648,32 +648,27 @@ trait EvalContextPrivExt<'mir: 'ecx, 'tcx: 'mir, 'ecx>: crate::MiriInterpCxExt<'
             };
 
             let (_size, _align, alloc_kind) = this.get_alloc_info(alloc_id);
```

Comment:
This does not care about size and align so it should call `is_alloc_live`, not `get_alloc_info`.
(We probably have more of these across Miri.)